template/docker/Dockerfile.j2

```diff
@@ -2,9 +2,13 @@
 # This file is automatically generated from the templates in stackabletech/operator-templating
 # DON'T MANUALLY EDIT THIS FILE
 # =============
-FROM oci.stackable.tech/sdp/ubi9-rust-builder AS builder
+# We want to automatically use the latest. We also don't tag our images with a version.
+# hadolint ignore=DL3007
+FROM oci.stackable.tech/sdp/ubi9-rust-builder:latest AS builder
 
-FROM registry.access.redhat.com/ubi9/ubi-minimal AS operator
+# We want to automatically use the latest.
+# hadolint ignore=DL3007
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest AS operator
```

Comment:
Actually I'd prefer to pin this and update via renovate but that's a separate PR :)

Reply:
Perhaps we should tag the rust-builder images with a semver, unless Renovate can find the SHA (or the short commit hash which is tagged)?

Do we need to create an issue for this, or will it surface naturally?

Reply:
I have an issue to "renovateify" everything....